### PR TITLE
Added Documentation for Trucker Job and Cargo Job

### PR DESCRIPTION
**Pull request by Taurusgirl252**
Added documentation for truck job

### CODE REVIEW

File: job/trucker-job.md [hunk 1]
```diff
@@ -1 +1,15 @@
...
\ No newline at end of file
+# Trucker Job
+
+> As a Trucker, your primary goal as a trucker is to transport goods to various destinations safe as possible; as This role requires a strong attention to detail, excellent driving skills, and a commitment to safety and customer service. Do you have what it takes to drive a big wheel or will you crash out before the end.
+
+## How to use the Trucker Job
+
+- Head to the trucker job at postal 10117.
+- Talk to the ped outside, and start work. 
+- Wait for about 5 minutes and a route will be available and click on view route and select your route
```
Comment (xSisyphusx) [inline job/trucker-job.md:9]: Please use exact times not "about"

File: job/trucker-job.md [hunk 2]
```diff
@@ -1 +1,15 @@
-# Trucker Job
\ No newline at end of file
+# Trucker Job
+
+> As a Trucker, your primary goal as a trucker is to transport goods to various destinations safe as possible; as This role requires a strong attention to detail, excellent driving skills, and a commitment to safety and customer service. Do you have what it takes to drive a big wheel or will you crash out before the end.
```
Comment (xSisyphusx) [inline job/trucker-job.md:3]: Clean up the language here a bit. "As a Trucker, your primary goal as a trucker" is a bit redundant.

File: job/trucker-job.md [hunk 3]
```diff
@@ -1 +1,15 @@
...
+
+## How to use the Trucker Job
+
+- Head to the trucker job at postal 10117.
+- Talk to the ped outside, and start work. 
+- Wait for about 5 minutes and a route will be available and click on view route and select your route
+- Select your truck
+- Drive to location on your GPS and pick up the trailer.
+- Drop off trailer at another location.
```
Comment (xSisyphusx) [inline job/trucker-job.md:12]: Be more specific here please

File: job/trucker-job.md [hunk 4]
```diff
@@ -1 +1,15 @@
...
+- Complete all routes and then return the vehicle back to the Trucker depot when done.
+
+- That's it, you now know how to drive a big rig. Great Job!
```
Comment (xSisyphusx) [inline job/trucker-job.md:15]: This is not necessary

**Review by xSisyphusx — CHANGES_REQUESTED** [review]
This is okay, but I need the language to be a little better.  Please see the review points below and resubmit the PR when completed.

File: job/trucker-job.md (hunk 2 above)
Comment (Taurusgirl252) [reply job/trucker-job.md:3]: Roger that will do.

File: job/trucker-job.md (hunk 1 above)
Comment (Taurusgirl252) [reply job/trucker-job.md:9]: Roger going back into it right now.

Comment (Taurusgirl252) [reply job/trucker-job.md:9]: I'm still new at this but I don't mind helping where it is needed. Thank you for the advice!

**Pull request merged**